test/smap-l2-gridder/smap-l2-gridder_Regression.ipynb

```diff
@@ -0,0 +1,298 @@
+    "        \"y-dim\": slice(1000, 2000),\n",
+    "        \"x-dim\": slice(8800, 9800),\n",
+    "    },\n",
+    "    \"Soil_Moisture_Retrieval_Data\": {\n",
+    "        \"y-dim\": slice(0, 1000),\n",
+    "        \"x-dim\": slice(2500, 3500),\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "non_production_configuration = {\n",
+    "    'SPL2SMP_E': {\n",
+    "        'collection_concept_id': Collection(id='C1268429712-EEDTEST'),\n",
+    "        'granule_id': 'G1268429718-EEDTEST',\n",
+    "        'selector': spl2smp_e_selector,\n",
+    "    },\n",
+    "    'SPL2SMAP': {\n",
+    "        'collection_concept_id': Collection(id='C1268429748-EEDTEST'),\n",
+    "        'granule_id': 'G1268429753-EEDTEST',\n",
+    "        'selector': spl2smap_selector,\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "\n",
+    "production_configuration = {\n",
+    "    'SPL2SMP_E': {\n",
+    "        'collection_concept_id': Collection(id='TBD'),\n",
+    "        'granule_id': '',\n",
+    "        'selector': spl2smp_e_selector,\n",
+    "    },\n",
+    "    'SPL2SMAP': {\n",
+    "        'collection_concept_id': Collection(id='TBD'),\n",
+    "        'granule_id': '',\n",
+    "        'selector': spl2smap_selector,\n",
+    "    },\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "37511561-96a4-4648-8a66-a8ce8730fd04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "environment_configuration = {\n",
+    "    'https://harmony.earthdata.nasa.gov': {\n",
+    "        'config': production_configuration,\n",
+    "        'env': Environment.PROD,\n",
+    "    },\n",
+    "    'https://harmony.uat.earthdata.nasa.gov': {\n",
+    "        'config': non_production_configuration,\n",
+    "        'env': Environment.UAT,\n",
+    "    },\n",
+    "    'https://harmony.sit.earthdata.nasa.gov': {\n",
+    "        'config': non_production_configuration,\n",
+    "        'env': Environment.SIT,\n",
+    "    },\n",
+    "    'http://localhost:3000': {\n",
+    "        'config': non_production_configuration,\n",
+    "        'env': Environment.LOCAL,\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "configuration = environment_configuration.get(harmony_host_url)\n",
+    "\n",
+    "if configuration is not None:\n",
+    "    harmony_client = Client(env=configuration['env'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "316542e3-0e6e-48d0-ad4d-dc02d20bdbe6",
+   "metadata": {},
+   "source": [
+    "### Run Tests\n",
+    "\n",
+    "* The cell below loops through the SMAP L2 collections for the configured environment.\n",
+    "* A request for gridding the L2 data is submitted to Harmony and the results downloaded.\n",
+    "* Verification compares the results to the a previously generated reference data file.  This comparison is done only on the first 2000 rows and 2000 columns of the downloaded data to save memory usage. \n"
```

Comment:
Thanks for adding these details - it helps.

Super nit: "...compares the results to ~~the~~ a previously generated..."

Reply:
[2bfc92e](https://github.com/nasa/harmony-regression-tests/pull/130/commits/2bfc92eccae724b672e1600a3649d56d7ed35b09)